microsite/blog/2020-08-05-announcing-backstage-software-templates.md

```diff
@@ -35,11 +35,11 @@ Since the templates can be customized to integrate with your existing infrastruc
 
 ### Golden Paths pave the way
 
-You can customize Backstage Software Templates to fit your organization’s standards. Using Go instead of Java? CircleCI instead of Jenkins? Serverless instead of Kubernetes? GCP instead of AWS? [Make your own recipes for any software component](/docs/features/software-templates/adding-templates.md) and your best practices will be baked right in.
+You can customize Backstage Software Templates to fit your organization’s standards. Using Go instead of Java? CircleCI instead of Jenkins? Serverless instead of Kubernetes? GCP instead of AWS? [Make your own recipes for any software component](https://backstage.io/docs/features/software-templates/adding-templates) and your best practices will be baked right in.
```

Comment:
Just checking if you meant to link to https://backstage.io/docs/features/software-templates/software-templates-index here?

Reply:
Thanks! Fixed.